dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
This will break dnf-plugins-core until that happens. And the last time I checked, that wasn't going to happen until this fall, in time for Fedora 25!

If there's something different, **please** tell me, because I need to know as the Mageia maintainer for the DNF stack.